include/octo-logger-cpp/manager.hpp

```diff
@@ -61,11 +63,14 @@ class Manager
     void configure(const ManagerConfigPtr& config, bool clear_old_sinks = true);
     void terminate();
     void stop(bool discard = false);
-    void dump(const Log& log, const std::string& channel_name, Logger::ContextInfo const& context_info);
+    void dump(const Log& log, const std::string& channel_name, ContextInfo const& context_info);
     void clear_sinks();
     void clear_channels();
     void restart_sinks() noexcept;
     const Logger& global_logger() const;
+    // @brief get the global context info
```

Comment:
remove brief, not really any useful info

Reply:
done